### PR TITLE
133 srahim dev

### PR DESCRIPTION
**Pull request by shahedrahim**

### CODE REVIEW

**Review by lorenzo-gomez-windhover — APPROVED** [review]
LGTM!

File: core/base/tools/commander/param_csv_config.yaml [hunk 1]
```diff
@@ -6,7 +6,9 @@ method:
   exportParameterValues
```
Comment (mbenson1) [inline core/base/tools/commander/param_csv_config.yaml:6]: What is the purpose of this file?  Is this to serve as a sample input file, could you add comments as such?

File: core/base/tools/commander/params_to_csv.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import argparse
```
Comment (mbenson1) [inline core/base/tools/commander/params_to_csv.py:2]: Add comments to this file describing what the code is doing, why, and how.

**Review by mbenson1 — CHANGES_REQUESTED** [review]

File: core/base/tools/commander/param_csv_config.yaml (hunk 1 above)
Comment (shahedrahim) [reply core/base/tools/commander/param_csv_config.yaml:6]: The file:
core/base/tools/commander/README.md
describes the purpose of the yaml file.  It is the file needed to configure the params pulled from YAMCS.

File: core/base/tools/commander/params_to_csv.py (hunk 2 above)
Comment (shahedrahim) [reply core/base/tools/commander/params_to_csv.py:2]: Added Comments in code

**Review by shahedrahim — COMMENTED** [review]
New changes on branch address comments made by reviewer.

File: core/base/tools/commander/params_to_csv.py (hunk 2 above)
Comment (shahedrahim) [reply core/base/tools/commander/params_to_csv.py:2]: fd680e96e8e9d4c0134cf825ae177ebe85931feb

**Pull request merged**